internal/injector/testdata/injector/http-no-arg-name/config.yml

```diff
@@ -37,6 +37,9 @@ aspects:
               {{ range .DirectiveArgs "dd:span" -}}{{printf "%q, %q," .Key .Value}}{{- end }}
             )
 
+import-map:
+  net/http: ""
```

Comment:
This also imports `log`... TBH I wonder if instead of specifying this by hand it could/should be grabbed from the AST as part of the test (parse for imports, then just harvest from the import specs).